packages/hvsock/hvsock.2.0.0/opam

```diff
@@ -0,0 +1,58 @@
+opam-version: "2.0"
+maintainer: "dave@recoil.org"
+authors: [ "David Scott" "Rolf Neugebauer" "Anil Madhavapeddy" "Simon Ferquel"]
+license: "ISC"
+homepage: "https://github.com/mirage/ocaml-hvsock"
+dev-repo: "git+https://github.com/mirage/ocaml-hvsock.git"
+bug-reports: "https://github.com/mirage/ocaml-hvsock/issues"
+doc: "https://mirage.github.io/ocaml-hvsock"
+
+build: [
+  [ "dune" "subst" ] {pinned}
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "base-bytes"
+  "base-threads"
+  "base-unix"
+  "lwt" {>= "3.2.0"}
+  "logs"
+  "fmt"
+  "cmdliner"
+  "sha"
+  "uri"
+  "base64" {>= "3.0.0"}
+  "uuidm"
+  "uutf"
+  "mirage-flow-lwt" {>= "1.2.0"}
+  "mirage-time-lwt" {>= "1.0.0"}
+  "cstruct" {>= "2.4.0"}
+  "duration"
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.2.0"}
```
Can you upstream the fix @djs55 ?

Reply:
Thanks! https://github.com/mirage/ocaml-hvsock/pull/65